Resources/Help/analyses/bayesianmetaanalysis.md

```diff
@@ -0,0 +1,158 @@
+Bayesian Meta-Analysis
```

Comment:
Any chance you could add the Dutch translation of this as well?
You can add the file in the same place, only name it bayesianmetaanalysis_nl.md

Reply:
Honestly, I am not sure what the right Dutch translations are for most terms. I could try to figure it out, but I really don't have time for that this week.

Reply:
I'll merge this before the Dutch translation, but it would be good if you could at least make a start with it and make a PR later. We don't have any dedicated translators to do this anymore unfortunately